source/compression/OP_COMPRESSED.rst

```diff
@@ -44,8 +44,8 @@ interpreted as described in `RFC 2119 <https://www.ietf.org/rfc/rfc2119.txt>`_.
 Terms
 =====
 
-*Compressor* - A compression algorithm, such as snappy or zlib.
-
+*Compressor* - A compression algorithm.  There are currently three supported
+algorithms: snappy, zlib and zstd.
```

Comment:
Can we throw an [Oxford comma](https://i.kym-cdn.com/entries/icons/original/000/017/771/the-oxford-comma_52c855ed979ed_w1500.jpg) in this list?